### DISCUSSION TITLE
Add a Summary page after the Abstract

### DISCUSSION THREAD

**Discussion by Miguel-azevedo9** [joaomlourenco/novathesis#466] [Q&A] [answered]
Professor,

I would like to add a Summary page after my abstract page, how can I do it?

Best Regards, 
Miguel Azevedo

**Comment by joaomlourenco** [comment, marked as answer]
Miguel, with LaTeX everything is possible…
But please notice that each University/School/Degree usually have specific rules about the look (format) and contents (what must be where) of their dissertations!
Are you sure you are allowed to add this "Summary" after the abstract?
